scene/2d/physics_body_2d.h

```diff
@@ -191,11 +192,32 @@ class RigidBody2D : public PhysicsBody2D {
 		ObjectID body_id;
 		ShapePair pair;
 	};
+	struct RigidBody2D_AddAction {
+		RID rid;
+		ObjectID id;
+		int shape = 0;
+		int local_shape = 0;
+		Vector2 local_pos;
+		Vector2 local_normal;
+		Vector2 local_vel_at_pos;
+		Vector2 collider_pos;
+		Vector2 collider_vel_at_pos;
+		Vector2 impulse;
+
+		RigidBody2D_AddAction() = default;
```

Comment:
```suggestion
		RigidBody2D_AddAction() {}
```
We use this syntax elsewhere in this file, and I don't think we use the `default` syntax anywhere

Reply:
It is used in various other parts the engine (e.g., the Vulkan renderer, the GDScript module), but does appear to be in the minority.

Reply:
You didn't format it as specified 🙂

Reply:
Yes, see the timestamp, I commented *before* you fixed this

Reply:
Ah, fair enough. Apologies for the double-push :)